snerk.py

```diff
@@ -501,8 +501,7 @@ def privmsg(self, sender, channel, msg):
             self.msg(channel, random.choice(sayings))
             return
 
-        elif (re.search(r'^\s*!\s*logs?\s*$', msg, re.I|re.U) or
-              re.search(r'^\s*logs?\s*!\s*$', msg, re.I|re.U)):
+        elif (re.search(r'(?<=\W)!logs?(?=\W)|(?<=\W)logs?!(?=\W)'ig, msg, re.I|re.U)
```

Comment:
I was trying to adapt https://regex101.com/r/cY6bP4/4 from matma, but I don't understand how the "ig" is meant to fit in... (Oh! Now I look afresh, I understand that the "ig" are covering how the 'test string' area gets parsed. I shall remove, and add the colon...)